app/models/document.rb

```diff
@@ -3,6 +3,8 @@
 class Document < ActiveFedora::Base
   include ::Hyrax::WorkBehavior
   include ::OregonDigital::DocumentMetadata
+  include ::OregonDigital::GenericMetadata
```

Comment:
included at the bottom of the file, which one should remain..remove one or the other